include/seqan3/alignment/pairwise/alignment_configurator.hpp

```diff
@@ -386,7 +386,12 @@ struct alignment_configurator
             }
             else if constexpr (align_ends_cfg_t::template is_static<1>())
             {
+#if defined(__GNUC__) && __GNUC__ >= 9
+                constexpr bool free_ends_trailing = align_ends_cfg_t::template get_static<1>();
+                return configure_edit_traits(std::integral_constant<bool, free_ends_trailing>{});
```

Comment:
We could also just omit the macro hackery here and keep this version, which is probably valid for all compilers. 
Also why is only this part affected? There is at least a second function with the same semantics. And this breaks nothing?

Reply:
We discussed, that we want to keep track of such bugs.